<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmlightbutton</class>
 <widget class="QWidget" name="frmlightbutton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>frmlightbutton</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="LightButton" name="lightbutton1" native="true"/>
   </item>
   <item>
    <widget class="LightButton" name="lightbutton2" native="true"/>
   </item>
   <item>
    <widget class="LightButton" name="lightbutton3" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LightButton</class>
   <extends>QWidget</extends>
   <header>lightbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
